src/bin/bat/clap_app.rs

```diff
@@ -430,7 +431,8 @@ pub fn build_app(interactive_output: bool) -> ClapApp<'static, 'static> {
                      '--style=\"..\"' option to the configuration file or export the \
                      BAT_STYLE environment variable (e.g.: export BAT_STYLE=\"..\").\n\n\
                      Possible values:\n\n  \
-                     * full: enables all available components (default).\n  \
+                     * default: enables basic style components (default).\n  \
```

Comment:
This is just a proposal, so I am mostly curious to hear what you think:

Maybe we should use the word "recommended" instead of "basic"?

```suggestion
                     * default: enables recommended style components (default).\n  \
```

Reply:
Sounds good, I remember that didn't quite feel right when I threw it in.